browser/BUILD.gn

```diff
@@ -140,7 +140,6 @@ source_set("browser_process") {
     "//brave/components/p3a",
     "//brave/components/resources",
     "//brave/components/services:brave_content_manifest_overlays",
-    "//brave/services/network/public/cpp",
```

Comment:
as discussed this should be fixed by making `//brave/services/network/public/cpp/BUILD.gn` a `component` or `jumbo_component`